### ISSUE TITLE
Missing git history

### ISSUE THREAD

**Issue by dxgldotorg** [ZeldaClassic/ZeldaClassic#15]
This repo was created in a manner that omitted all commit history previously available from the other repo.

**Comment by Gleeok** [comment]
Yes, it was done on purpose for this specific branch. I'll wait to reply again until we have the history and historical branches up. We're still shuffling things around in the meantime.

**Comment by dxgldotorg** [comment]
Using `git clone --mirror` can do wonders.
